packages/cozy-mespapiers-lib/src/components/Views/Home.jsx

```diff
@@ -38,13 +40,12 @@ const Home = () => {
   )
 
   const contactIds = getContactsRefIdsByFiles(papers)
-  const contactsQueryByIds = buildContactsQueryByIds(contactIds)
+  const contactsQuery = flag('mespapiers.flexsearch.enabled')
+    ? buildContactsQuery(!isLoadingFiles)
+    : buildContactsQueryByIds(contactIds, !isLoadingFiles)
```

Comment:
Don't that mean that if the search is enabled, we will always have an unecessary contact query that will get all the contacts ids?

Reply:
non ce n'est pas une requête c'est une fonction synchrone, mais effectivement inutile dans le cas de flexsearch, je corrige.